infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -31,6 +34,11 @@ export class AtatWebApiStack extends cdk.Stack {
       value: table.tableName,
     });
 
+    const submitQueue = new sqs.Queue(this, "SubmitQueue", {
+      queueName: "SubmitQueue.fifo",
+      fifo: true,
```

Comment:
Can we design our API such that we don't need a FIFI?

Reply:
What is a FIFI?
Do you mean FIFO?

Reply:
lol yeah, definitely meant FIFO